src/applications/representative-form-upload/containers/WithAccessTokenManager.jsx

```diff
@@ -0,0 +1,40 @@
+import { useEffect } from 'react';
+import api from '../utilities/api';
+
+const EXPIRATION_TIME = 3 * 60 * 1000; // 3 minutes
+
+const WithAccessTokenManager = ({ userLoggedIn, children }) => {
```

Comment:
Suggest dropping the `With`. Maybe just me, but my brain associates `With` with higher-order components.